src/react-components/buttons/radio-button.jsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * This program is part of the OpenLMIS logistics management information system platform software.
+ * Copyright © 2017 VillageReach
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *  
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+ * See the GNU Affero General Public License for more details. You should have received a copy of
+ * the GNU Affero General Public License along with this program. If not, see
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org. 
+ */
+
+import React from 'react';
+
+export const RadioButton = ({ changed, id, isSelected, label, additionalInfo="", value, disabled }) => {
+    return (
```

Comment:
If component only returns a view, without any logic or extracting necessary date before return, you cna use implicit return and omit return keyword

`const RadioButton = ({ changed, id, isSelected, label, additionalInfo="", value, disabled }) => (
    <div className="RadioButton"> ... </div>
)`